src/browser/web-view.js

```diff
@@ -751,6 +764,10 @@ const viewFrame = (model, address) =>
     onBlur: on(address, always(BlurShell)),
     onFocus: on(address, always(FocusShell)),
     // onMozbrowserAsyncScroll: on(address, decodeAsyncScroll),
+    // @TODO send force changed actions and update webviews animation code
+    // to respond live 1:1 with push.
+    // onServoMouseForceChanged: on(address, decodeForceChanged),
+    onServoMouseForceDown: on(address, decodeForceDown),
```

Comment:
Is there reason to not put this into `html.div` returned by `view` function instead ? It's just we already have too event many handlers on iframe itself & I'd prefer not add more unless they are necessary of course.